docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -32,21 +43,45 @@ The kubectl plugin lines use fully specific executables (e.g.,
 `kubectl kubestellar prep-for-syncer` corresponds to
 `bin/kubectl-kubestellar-prep_for_syncer`).
 
+### Initialize the KubeStellar platform
+
+In this step KubeStellar creates and populates the Edge Service
+Provider Workspace (ESPW), which exports the KubeStellar API, and also
+augments the `root:compute` workspace from kcp TMC as needed here.
+Those augmentation consists of adding authorization to update the
+relevant `/status` and `/scale` subresources (missing in kcp TMC) and
+extending the supported subset of the Kubernetes API for managing
+containerized workloads from the four resources built into kcp TMC
+(`Deployment`, `Pod`, `Service`, and `Ingress`) to the other ones that
+are namespaced and are meaningful in KubeStellar.
+
+```shell
+kubestellar init
+```
+
+### Create an inventory management workspace.
+```shell
+kubectl ws root
+kubectl ws create imw-1 --enter
+```
 ### Create SyncTarget and Location objects to represent the florin and guilder clusters
 
 Use the following two commands. They label both florin and guilder
 with `env=prod`, and also label guilder with `extended=si`.
 
 ```shell
+kubectl ws root:imw-1
 kubectl kubestellar ensure location florin  loc-name=florin  env=prod
 kubectl kubestellar ensure location guilder loc-name=guilder env=prod extended=si
+echo "decribe the florin location object"
+kubectl describe location.edge.kcp.io florin
```

Comment:
`get -o yaml` would be better than`describe` here.  There are no associated events (which `describe` would display if there were any), and the Managed Fields (which are displayed by `describe` but not normally by `get`) are verbose and not valuable here.